app/javascript/components/ServiceForm.js

```diff
@@ -132,10 +134,17 @@ class ServiceForm extends React.Component {
           </div>
           <ul className="form-actions__items">
             <li className="form-actions__item">
-              <input type="submit" value="登録" name="commit" />
+              <Button
+                onClick={() => { document.getElementById('service-form').dispatchEvent(new Event('submit')); }}
+                color="primary"
+                size="md"
+                block="block"
```

Comment:
[IMO] block という props に "block" という文字列を渡すというインターフェースはちょっと違和感を感じるかもです 😺 
string だと何種類もありそうにも見えてしまいますし😁 
この props は「ボタンが block であるかそうでないか」という 2種類の状態を切り替える props なので、↓のように boolean にしてみてはどうでしょう〜？ 😄 

```suggestion
                isBlock={true}
```

Reply:
ありがとうございます!
確かに、2種類の状態しかないのでbooleanを受け取るようにする方が良いですね。

デフォルトではfalseで、ButtonコンポーネントにisBlockが指定されたときにclass属性に`btn--block`が付与されるように修正しました。j